### ISSUE TITLE
Broken with newer version of Firefox

### ISSUE THREAD

**Issue by MassimoLauria** [bnbeckwith/pintaboard#8]
When I start a window it says:
`Sourcing file: /Users/massimo/.pentadactyl/plugins/pintaboard.js:147: SyntaxError: missing ] after element list`

Firefox 46.0.1
Pentadactyl: 1.2pre (actually from GitHub)

**Comment by granin** [comment]
Anyone can reproduce this?

**Comment by MassimoLauria** [comment]
To reproduce this I am running

 - Firefox 47.0
 - Pentadactyl git master branch   hash=1a4290d92ac3e2dfd3b84168081ba721c02556b0
 - Mac OSX El Capitan

And the problem occurs immediately after startup even using the minimal configuration file
```
loadplugins '\.(js|penta)$'
set pintaboardToken="************************"
```
where of course I substitute the asterisks with my token.

**Comment by bnbeckwith** [comment]
This is fixed with commit 47ee795
